pkg/ingester/active_series_test.go

```diff
@@ -8,27 +8,33 @@ import (
 	"sync"
 	"testing"
 	"time"
+	"unsafe"
 
 	"github.com/prometheus/prometheus/model/labels"
 	"github.com/stretchr/testify/assert"
 )
 
 func copyFn(l labels.Labels) labels.Labels { return l }
 
+func FromLabelAdaptersToLabels(ls []labels.Label) labels.Labels {
+	return *(*labels.Labels)(unsafe.Pointer(&ls))
+}
```

Comment:
cortexpb.FromLabelAdaptersToLabels only handle a []LabelAdapter labels, this may change current tests' variables, so a new local FromLabelAdaptersToLabels seems better suitable.